From 2c5506f4ed36ca664e128158aeb17fa970d130d0 Mon Sep 17 00:00:00 2001
From: Till Kamppeter <till.kamppeter@gmail.com>
Date: Tue, 4 Oct 2016 11:44:01 +0200
Subject: Removed the lines in the UDEV script for the automatic firmware

 upload into the printer which remove the UDEV rules files for HPLIP's
 automatic firmware upload.
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/foo2zjs/+bug/783389
---
 hplj1000 | 10 ----------
 1 file changed, 10 deletions(-)

diff --git a/hplj1000 b/hplj1000
index 6b74601..6bc0d18 100755
--- a/hplj1000
+++ b/hplj1000
@@ -152,16 +152,6 @@ case "$0" in
 esac
 
 #
-#	Remove HPLIP proprietary rules!
-#
-model=` echo "$MODEL" | tr 'A-Z' 'a-z' `
-rm -f /etc/udev/rules.d/*hpmud*laserjet_${model}*
-rm -f /etc/udev/rules.d/*hpmud_support.rules
-rm -f /etc/udev/rules.d/*hpmud_plugin.rules
-rm -f /lib/udev/rules.d/*hpmud_support.rules
-rm -f /lib/udev/rules.d/*hpmud_plugin.rules
-
-#
 #	The special command line argument "install-usermap" will install
 #	the proper entry into the /etc/hotplug/usb.usermap file
 #
